### PR TITLE
feat: Change installation namespace

### PR DESCRIPTION
**Pull request by jaideepr97**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

> /kind bug
> /kind cleanup
> /kind failing-test
/kind enhancement
> /kind documentation
> /kind code-refactoring

**What does this PR do / why we need it**:
Changes required for gitops-operator to be installed in its own namespace out of the box 
Following changes are included:
- change default namespace to `openshift-gitops-operator`
- add cluster monitoring to new namespace
- update control-plane label on all operator resources 

**Have you updated the necessary documentation?**

* [x] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**Test acceptance criteria**:

* [ ] Unit Test
* [ ] E2E Test

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ spec:
 
 ![a relative link](docs/assets/operatorhub-listing.png)
 
-3. Install the operator using the defaults in the wizard, and wait for it to show up in the list of "Installed Operators". If it doesn't install properly, you can check on its status in the "Installed Operators" tab in the `openshift-operators` namespace.
+3. Install the operator in the `openshift-gitops-operator` namesapce using the defaults in the wizard, and optionally, select the checkbox to enable cluster monitoring on the namesapce. Wait for it to show up in the list of "Installed Operators". If it doesn't install properly, you can check on its status in the "Installed Operators" tab in the `openshift-gitops-operator` namespace.
```
Comment (Srivaralakshmi) [inline README.md:30]: @jaideepr97 Do we have any possible installation failures and underlying causes or symptoms for the same? If yes, can we have troubleshooting tips for them? Thanks!

Comment (Srivaralakshmi) [reply README.md:30]: @jaideepr97 I see that the user has to just select/enable the OpenShift monitoring option while installing the GitOps Operator in the `openshift-gitops-operator` namespace.

- Do we need any documentation or information like a concept/overview, or anything that we need in the Downstream documentation from the user's perspective, about monitoring metrics?

Comment (Srivaralakshmi) [reply README.md:30]: @jaideepr97 Can you please respond to my queries here? Thanks!

Comment (jaideepr97) [reply README.md:30]: @Srivaralakshmi  I don't know what the best place to put this info is so I'm just putting it here :


Installation instructions to be able to use this feature:
- fresh installations of the operator will see the option to choose the namespace to install the operator in, with the recommended option already selected as `openshift-gitops-operator` namespace 
- users must select the check box saying `Enable Operator recommended cluster monitoring on this Namespace` to enable cluster monitoring 

ns-installation




Usage instructions:
- Once operator is installed, users can leverage the  `Observe > metrics` page in the admin view to query the metrics exposed by the operator 
- At present the new operator performance metrics exposed are:
 - `active_argocd_instances_total` [_Guage_] - This metric produces the graph that tracks the total number of active argo-cd instances being managed by the operator at a given time
 - `active_argocd_instances_by_phase{phase=<phase>}` [_Guage_] - This metric produces the graph that tracks the count of active Argo CD instances by their phase [Available/Pending/Failed/unknown]
 - `active_argocd_instance_reconciliation_count{namespace="<argocd-instance-ns>"}` [_Counter_] - This metric produces the graph that tracks total number of reconciliations that have occurred for the instance in the given namespace at any given point in time
 - `controller_runtime_reconcile_time_seconds_per_instance_bucket{namespace="<argocd-instance-ns>",le="0.5"}` [_Histogram_]- This metric tracks the number of reconciliations that took under 0.5s to complete for a given instance
 
 Any of these metrics can be queried against prometheus as described above, or can be used for more complex queries as is true with any other promql queries 

reconciliations_per_instance
argocd_instances_by_phase
active_argocd_instances_total



>

**Comment by iam-veeramalla** [comment]
@jaideepr97 can you create the operator bundle and verify if the operator upgrade is working fine ?

**Comment by iam-veeramalla** [comment]
CI failures are expected as `openshift/release` repo should be updated for the CI changes to support this PR.

PR for `openshift/release` repo is created and will be merged once this PR is tested and passing the CI.
https://github.com/openshift/release/pull/43023

**Pull request merged**